lib/hardware/buses/iob_wishbone2iob/iob_wishbone2iob.py

```diff
@@ -3,11 +3,348 @@ def setup(py_params_dict):
+                "signals": [
+                    {
+                        "name": "iob_rvalid",
+                        "width": 1,
+                        "direction": "input",
+                    },
+                ],
+            },
+            {
+                "name": "iob_rdata_i",
+                "descr": "Input port",
+                "signals": [
+                    {
+                        "name": "iob_rdata",
+                        "width": "DATA_W",
+                        "direction": "input",
+                    },
+                ],
+            },
+            {
+                "name": "iob_ready_i",
+                "descr": "Input port",
+                "signals": [
+                    {
+                        "name": "iob_ready",
+                        "width": 1,
+                        "direction": "input",
+                    },
+                ],
+            },
+            {
+                "name": "wb_ack_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "wb_ack",
+                        "width": 1,
+                        "direction": "output",
+                    },
+                ],
+            },
+            {
+                "name": "wb_data_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "wb_data",
+                        "width": "DATA_W",
+                        "direction": "output",
+                    },
+                ],
+            },
+            {
+                "name": "iob_valid_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "iob_valid",
+                        "width": 1,
+                        "direction": "output",
+                    },
+                ],
+            },
+            {
+                "name": "iob_address_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "iob_address",
+                        "width": "ADDR_W",
+                        "direction": "output",
+                    },
+                ],
+            },
+            {
+                "name": "iob_wdata_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "iob_wdata",
+                        "width": "DATA_W",
+                        "direction": "output",
+                    },
+                ],
+            },
+            {
+                "name": "iob_wstrb_o",
+                "descr": "Output port",
+                "signals": [
+                    {
+                        "name": "iob_wstrb",
+                        "width": "DATA_W/8",
+                        "direction": "output",
+                    },
+                ],
+            },
```

Comment:
"iob" is a standard interface that can be generated with `if_gen.py`. There is no need to create signals one by one.
"wishbone" is a standard interface but cannot be currently generated with `if_gen.py`. However, we may still group its signals in the same port.
```suggestion
            {
                "name": "iob",
                "interface": {
                    "type": "iob",
                    "subtype": "master",
                    "ADDR_W": "ADDR_W",
                    "DATA_W": "DATA_W",
                },
                "descr": "IOb native master interface",
            },
            {
                "name": "wb",
                "descr": "Wishbone slave interface",
                "signals": [
                   {
                        "name": "wb_addr",
                        "width": "ADDR_W",
                        "direction": "input",
                    },
                    {
                        "name": "wb_select",
                        "width": "DATA_W/8",
                        "direction": "input",
                    },
                    {
                        "name": "wb_we",
                        "width": 1,
                        "direction": "input",
                    },
                    {
                        "name": "wb_cyc",
                        "width": 1,
                        "direction": "input",
                    },
                    {
                        "name": "wb_stb",
                        "width": 1,
                        "direction": "input",
                    },
                    {
                        "name": "wb_data",
                        "width": "DATA_W",
                        "direction": "input",
                    },
                    {
                        "name": "wb_ack",
                        "width": 1,
                        "direction": "output",
                    },
                    {
                        "name": "wb_data",
                        "width": "DATA_W",
                        "direction": "output",
                    },
                ],
            },
```